data/campaigns/Under_the_Burning_Suns/utils/abilities.cfg

```diff
@@ -587,6 +587,17 @@ Any units adjacent to this unit will fight as if it were dusk when it is night,
                     find_in=utbs_tailwind2_sources
                 [/filter_adjacent]
             [/filter]
+            # Temporarily increase the max moves correspondingly, so the unit gets a green orb and
+            # rest-heals next turn. Using `duration=turn` means that the wrong max moves will show
+            # during the AI turn, but using `duration=turn end` allows the unit to move 1 or 2 hexes
+            # and still rest-heal.
```

Comment:
Is the idea here to not allow moving 1 or 2 hexes and still rest heal? Is this a workaround or the final fix?

To me allowing rest heal even if using the bonus moves seems more sensible.

Reply:
@Hejnewar I think this is a question for you, about what you want the campaign to do.

My feeling is that the 2 extra hitpoints are for avoiding putting strain on the wounds by not moving at all. Okay, so it's also given if the unit defended themselves, but that's WINR.

Reply:
On one hand I really like the green orb. On the other hand it makes it harder to see which units got buffed. On the third hand, future proofing the game is good, unless we agree that abilities that increase movement should always also increase max moves as well but we can't enforce that in add-ons. I'm generally fine with either take, it is small enough difference that it's fine.  

When it comes to lore then for this particular unit it already heals as well so we can actually probably just assume that mystic fixed the wounds enough that nothing will happen if the unit moves a bit, but it only works in this case.

Reply:
Latest push has changed it so that rest healing still happens even after using the bonus moves.